scrapy/crawler.py

```diff
@@ -56,6 +57,7 @@ def __init__(
         spidercls: Type[Spider],
         settings: Union[None, dict, Settings] = None,
         init_reactor: bool = False,
+        addons: Optional[AddonManager] = None,
```

Comment:
Why does Crawler need this argument? It doesn't accept e.g. a SignalManager or StatsCollector.